man/systemd-coredump.xml

```diff
@@ -58,59 +58,77 @@
 
   <refsect1>
     <title>Description</title>
+    <para><command>systemd-coredump</command> is a system service that can acquire core dumps
+    from the kernel and handle them in various ways.</para>
 
-    <para><command>systemd-coredump</command> can be used as a helper
-    binary by the kernel when a user space program receives a fatal
-    signal and dumps core. For it to be used in this capacity, it must
-    be specified by the
-    <varname>kernel.core_pattern</varname> <citerefentry project='man-pages'><refentrytitle>sysctl</refentrytitle><manvolnum>8</manvolnum></citerefentry>
-    setting. The syntax of this setting is explained in
-    <citerefentry project='man-pages'><refentrytitle>core</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
-    Systemd installs <filename>/usr/lib/sysctl.d/50-coredump.conf</filename> which configures
-    <varname>kernel.core_pattern</varname> to invoke <command>systemd-coredump</command>.
-    This file may be masked or overridden to use a different setting following normal
-    <citerefentry><refentrytitle>sysctl.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>
-    rules.</para>
-
-    <para>The behavior of a specific program upon reception of a
-    signal is governed by a few factors which are described in detail
-    in <citerefentry project='man-pages'><refentrytitle>core</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
-    In particular, the coredump will only be processed when the
-    related resource limits are high enough. For programs started by
-    <command>systemd</command>, those may be set using
-    <varname>LimitCore=</varname> (see
-    <citerefentry><refentrytitle>systemd.exec</refentrytitle><manvolnum>5</manvolnum></citerefentry>).
+    <para>Core dumps can get written to the journal and saved as a file. Once saved they can e. g.
```

Comment:
Also "get" doesn't sound right. "Core dumps can be written to the journal or saved as a file. Once saved they can be retrieved for further processing, for example in \<>gdb\</>".